app.js

```diff
@@ -42,78 +42,85 @@ const port = 8081;
 
 //LOAD CONFIG.ENV VARS
 dotenv.config({
-	path: './config/config.env'
+  path: "./config/example-config.env",
```

Comment:
This should be config.env

Reply:
Yeah, will fix that. I was switching between config and example. We should handle with environment variables :thinking:

Reply:
I added a local option for scripts (allows for easier switching between environments). Did that rather than using dev since dev requires local mongoDB instance.